5.x/crud-operation-trash.md

```diff
@@ -0,0 +1,154 @@
+# Trash Operation <span class="badge badge-info">PRO</span>
+
+--
+
+<a name="about"></a>
+## About
+
+This CRUD operation allows your admins to soft delete, restore and permanently delete entries from the table.
+
+<a name="requirements"></a>
+## Requirements
+
+This is a <span class="badge badge-info">PRO</span> operation. It requires that you have [purchased access to `backpack/pro`](https://backpackforlaravel.com/products/pro-for-unlimited-projects).
+
+In addition, it needs:
+- Your entity must use Laravel's `SoftDeletes` trait, for this Operation to work
+
+<a name="trash-a-single-item"></a>
+## Trash a Single Item <span class="badge badge-info">PRO</span>
+
+<a name="how-it-works"></a>
+### How it Works
+- **Trash**:
+Using AJAX, a DELETE request is performed towards ```/entity-name/{id}/trash```, which points to the ```trash()``` method in your EntityCrudController.
+
+- **Restore**:
+Using AJAX, a PUT request is performed towards ```/entity-name/{id}/restore```, which points to the ```restore()``` method in your EntityCrudController.
+
+- **Delete**:
+Using AJAX, a DELETE request is performed towards ```/entity-name/{id}/delete-permanently```, which points to the ```deletePermanently()``` method in your EntityCrudController.
+
+<a name="enabling"></a>
+### How to Use
+
+To enable it, you need to ```use \Backpack\PRO\Http\Controllers\Operations\TrashOperation;``` on your EntityCrudController. For example:
```

Comment:
Actually it won't work if you try to use `Backpack\PRO`, you have to use `Backpack\Pro` (not all caps). Took me more time to figure this out than I would like to admit 😅

```suggestion
To enable it, you need to ```use \Backpack\Pro\Http\Controllers\Operations\TrashOperation;``` on your EntityCrudController. For example:
```